packages/arb-ts/package.json

```diff
@@ -41,15 +43,20 @@
     "@ethersproject/bignumber": "^5.1.1",
     "@ethersproject/bytes": "^5.0.8",
     "@nomiclabs/hardhat-ethers": "^2.0.2",
-    "@types/prompts": "^2.0.14",
+    "@typechain/ethers-v5": "^7.2.0",
+    "arb-bridge-eth": "0.7.2",
+    "arb-bridge-peripherals": "1.0.0",
+    "arbos-precompiles": "1.0.1",
     "dotenv": "^10.0.0",
     "ethers": "^5.1.0",
-    "prompts": "^2.4.2"
+    "ts-node": "^10.2.1"
```

Comment:
can't this stay in dev dependencies? i.e., only additionally used in install hooks, no?

Reply:
we use ts-node in the psotinstall hook. if this is consumed as a lib, the devdependency won't be installed.
an alternative is to actually compile it with tsc and run with node